scripts/test/Muon/FFTPresenter_test.py

```diff
@@ -35,7 +35,8 @@ def setUp(self):
         self.view.getFFTRePhase=mock.Mock()
         self.view.getFFTImPhase=mock.Mock()
         self.view.getWS=mock.Mock(return_value="MUSR00023456")
-        self.view.getAxis=mock.Mock(return_value="x")
+        self.view.getFirstGoodData=mock.Mock(return_value=0.1)
```

Comment:
The asserts and assignments in this file would also be much more readable with a few extra spaces